README.md

```diff
@@ -1,88 +1,127 @@
-The user sees a login screen which
-accepts a username based on either a traveler or an agent of a travel agency.
-When a traveler logs in a fetch call is made to the API to verify the travelers
-ID against the ID's in the Array of travelers.
+- [Overview](#overview)
+- [Screenshot](#screenshot)
+- [Setup](#setup)
+- [Tech Stack](#tech-stack)
+- [Learning Objectives](#learning-objectives)
+- [Credits](#credits)
+- [Contact](#contact)
 
-#### Traveler Dashboard
+# Overview
 
-**If a traveler logs in they see a dashboard page showing them:**
+Travel Tracker utilizes the fetch API to gather data from multiple endpoints. A user is able to login as either an agent or traveler, and then they are able to view trips, make trip requests, and approve / delete trip requests depending on who is logged in.
 
-- All trips taken (past, present, upcoming, and pending)
-- Total amount of money spent on trips this year (This
-  is calculated from the trips data and includes a 10%
-  agent fee)
+#### To login as an agent use the following login information:
 
-#### Agent Dashboard
+    Username: agency
+    Password: travel2020
 
-**If an agent logs in they see a dashboard page showing them:**
+####As an agent who is logged in you will:
 
-- New trip requests (a users pending trips)
-- Total income generated this year (will be 10% of user
-  trip cost)
-- Travelers on trips for today's date
+- See any pending trips
+- Be able to approve any pending trips
+- See any trips happening on today's date
+- Be able to search for a traveler and then approve or delete any trips for that traveler
 
-#### Traveler Interaction
+#####To approve a pending trip:
+Locate the trip you would like to approve in the PENDING TRIPS section
+Click APPROVE
+You will see a SUCCESS message
+The trip will no longer be in the PENDING TRIPS section upon refreshing and logging back in as agent OR logging out and logging back in as agent
+#####To search for a traveler and approve / delete pending trip:
+Search for the traveler by name in the search field
+Click SHOW TRAVELER
+Any PENDING trips will be shown first in the list of this travelers trips
+To APPROVE the trip, click APPROVE
+To DELETE the trip, click DELETE
+Depending on the choice above the trip will now be showing as either APPROVED in the travelers list of UPCOMING TRIPS or will no longer be showing at all if it was DELETED
 
-**A traveler who is logged in is able to make a trip request:**
+#### To login as any one of 50 travelers use the following login information:
 
-- Select a date, duration, number of travelers
-- Choose from a list of destinations
-  After making these selections, they see an estimated
-  cost (including a 10% travel agent fee) for the trip.
-- Once they submit the trip request, it will show on
-  their dashboard as “pending” so that the travel agency
-  can approve or deny it.
+    Username: traveler1  ( 1 represents the TravelerID and can be any number from 1 - 50 )
```

Comment:
maybe instead of any number 1-50, it is "Any valid traveler ID from the database" and you reference the endpoint to see the list of valid id's?